pkg/registries/registries.go

```diff
@@ -238,8 +353,10 @@ func EditRegistriesConfig(config *sysregistriesv2.V2RegistriesConf, insecureScop
 			}
 		}
 	}
+
+	mirrorSets := append(icspMirrorSets, append(digestMirrorSets, tagMirrorSets...)...)
```

Comment:
Non-blocking: 
```suggestion
	allMirrorSets := append(icspMirrorSets, append(digestMirrorSets, tagMirrorSets...)...)
```
to be a bit more explicit here